scripts/run_saucelabs.js

```diff
@@ -109,19 +147,29 @@ async function runKarma(testFile) {
  * of all child processes.  This allows any failing test to result in a CI
  * build failure for the whole Saucelabs run.
  */
-async function runNextTest(maxExitCode = 0) {
+async function runNextTest(maxExitCode = 0, results = {}) {
   // When test queue is empty, exit with code 0 if no tests failed or
   // 1 if any tests failed.
-  if (!testFiles.length) process.exit(maxExitCode);
+  if (!testFiles.length) {
+    for (const fileName in results) {
```

Comment:
Nit: `for (const fileName of Object.key(results))` to include only OwnProperties.

Reply:
Done.